packages/js/product-editor/src/components/number-control/number-control.tsx

```diff
@@ -71,10 +86,59 @@ export const NumberControl: React.FC< NumberProps > = ( {
 				{ ...inputProps }
 				step={ step }
 				disabled={ disabled }
+				autoComplete="off"
 				id={ id }
-				suffix={ suffix }
+				className="woocommerce-number-control"
+				suffix={
+					<>
+						{ suffix }
+						{ isFocused && (
+							<>
+								<Button
+									className="woocommerce-number-control__increment"
+									icon={ plus }
+									onClick={ () =>
```

Comment:
It would be nice to change this to onMouseDown and use a timer to allow the user to quickly step through values by holding down the mouse button, as the native stepper does.

https://stackoverflow.com/questions/71456174/repeat-the-function-with-onmousedown-in-react